### ISSUE TITLE
Feature Request - Add verification statuses to agent profiles

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7235]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

Some agents are curated and the people doing the curating would like to indicate that. Some agents are not necessarily curated, but contain appropriate information to make them "community curated". In both cases, it would be nice to indicate to users that this is a known agent with some metadata that can be used to disambiguate the agent from others with the same or similar names. It would also be nice to ensure that the curated agents are not vandalized and are used appropriately in association with data that actually should be associated with them.

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

Provide a barrier to help ensure that anyone editing curated agents will not do so lightly - perhaps even consulting with a known agent curator before making changes. Also, provide a method for users of Arctos data to assess the quality of an Agent profile.

**Describe the solution you'd like**
_How might we accomplish your goals?_

A system, perhaps similar to the one we use for verification of georeferences that allows an Arctos agent to take responsibility for curation of an agent or allows Arctos to flag an agent as containing data that provides verification (generally an external identifier (ORCiD, Wikidata, Library of Congress).

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

Using a relationship "curated by" to indicate curated agents (this could be completed with a bot agent for automated "curation").

Creating a new "thing" that allows for marking an agent as verified (curated or managed by an Arctos collection), accepted (contains supporting metadata), or unverified (an agent with no supporting metadata)

**Additional context**
_Add any other context or screenshots about the feature request here._

This suggestion came from the Arctos Agents meeting today - https://docs.google.com/document/d/1-nssmG4VRJFP8Ek1wPDzkfpDoqLMs3eR/edit#heading=h.iga5l75xe9bw 

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

Adding wildfire priority because this is part of larger Agent discussions that cannot proceed until we have a solution here.

@droberts49 @wellerjes please chime in!

**Comment by dustymc** [comment]
I like relationships for this.

* it provides an "owner"
* The "owner" can be anything - an institution, a collaboration, a person, "some institution's avoid-at-all-costs anti-curation agent", WHATEVER.
* The "owner" cannot be nothing, and avoiding "THIS IS SUPER IMPORTANT (according to some undisclosed entity at some time for some unknown purpose probably)" seems important here.
* There's metadata (it's easy to find my oopsies, just follow ``created_by_agent_id``)
* There's a date component, which seems super handy for eg figuring out if the mess was before or after the 'claim'
* it's expandable - 'unverified' (or whatever) is just another relationship (that is, it's data, not structure)
* It should seamlessly export to a more attribute-like structure, if we go there.
* It's as good a trigger as anything else - this can be used to fire off any sort of 'business logic' regarding the agent. (We already have such triggers eg around 'bad duplicate of' agents - there's even an existing recipe.)
* It can happen NOW - someone file a request for a new relationship, ping we box-checkers, this can happen today.

Alternatively, let me know here what relationships cannot do, we'll use that to shape https://github.com/ArctosDB/arctos/issues/5172 and/or https://docs.google.com/spreadsheets/d/1h0cxNvMyIJuM9M30g75t5hG7k6V1FbxEOlUb7c7cnvA/edit#gid=0

>or allows Arctos to flag an agent as containing data that provides verification 

Maybe I'm just lost in the weeds, but ANYTHING we might do locally looks like a very poor replacement for an association with an external identifier. (Wikidata has orders of magnitude more eyeballs than Arctos ever will, ORCID requires involvement of the person, etc.) I would encourage not mixing those up (but nothing about the relationship model prevents such things if there's some reason to do so).

> completed with a bot agent

As a user, I think I would 100% ignore anything asserted by a bot. Anything worth knowing should be supported by better data (eg I think perhaps you're suggesting that an ORCID might fire off the bot - but then someone could remove the ORCID....), and anything not supported by better data - well, I don't get it, but if there's some reason we do need this then the metadata the relationship model carries probably makes it make sense.

>wildfire priority

Agreed, this should come before or at the same time as https://github.com/ArctosDB/arctos/issues/7135 / https://docs.google.com/document/d/1tSk344mJsIRYZ-BJcYfyaPEmbrFS-wSqZzJ5dO1E2EU/edit - let's do whatever it takes to resolve this while it's still fresh and before the landscape changes in any dramatic ways.

**Comment by wellerjes** [comment]
Thanks @Jegelewicz!

We were envisioning something like this on the Agents page, similar to the Event verification: 
![image](https://github.com/ArctosDB/arctos/assets/93737929/e0296e10-7844-424b-ba93-765581c9d2f6)

With the relationships model, would it be possible to have it appear near the top of the page? Agents pages for institutions are already long with the number of employees, volunteers, historic collectors, donors, etc. I'm concerned about the verification data getting lost in the long list of relationships [(see CHAS profile for reference)](https://arctos.database.museum/agent/21293310).

>but ANYTHING we might do locally looks like a very poor replacement for an association with an external identifier
Many of our agents don't have Wikidata pages because the information we are adding to the record is coming directly from our own archives/records. Agents without Wikidata/ORCID/LOC etc. should not be considered of lesser quality because these links don't exist. Showing that the agent is managed by CHAS based on archives/records should give it similar weight to having those IDs.

>I think I would 100% ignore anything asserted by a bot.
Agreed. My preference would be to verify the record myself, or assign to a volunteer or intern. I'm going to want a set of eyes on them regardless, but I realize not everyone has that preference.

**Comment by dustymc** [comment]
>possible to have it appear

"just UI"

>something like this on the Agents page

That's one relationship minus ``The "owner" cannot be nothing`` - they can in that model, which I'd think would be critical for things like....

>should not be considered of lesser quality

I've just seen way too much data which has and has not been exposed to daylight to preemptively buy into that; I haven't actually tried to do anything with your data and maybe I'll change my mind when I do, but even then you're (we hope) not the only one doing this sort of thing and I'll be suspect of the data from XYZ until it proves itself. I very strongly suspect a simple "this good" flag just won't be useful for many users, where something with the depth relationships bring would.

I might also put more trust into data which has been 'curated' (whatever we call this) by both you and XYZ - yet another task that relationships accommodate easily and a simple flag just can't.

>verify the record myself, or assign to a volunteer or intern

EXACTLY! Maybe the data will quickly teach me that THIS intern and THAT volunteer produce data of different qualities and I want to sort them into separate piles, where some other user (such as yourself) might want all "children of CHAS" (yet another place where the depth is handy - relationships can lead to relationships) in one category.

Still not seeing anything relationships fail to accommodate.

**Comment by Jegelewicz** [comment]
> Still not seeing anything relationships fail to accommodate.

Agree - we would just need to ask for the UI changes to agent pages and possibly something else - any agent bearing a relationship of "verified by" would require an additional step to edit - an acknowledgement by the editor that they are editing an agent that someone is curating and that they are making the edit with the confidence that it will increase the quality of the agent data. I can make the code table request and we can discuss there?

**Comment by dustymc** [comment]
>acknowledgement

This is all a little theoretical at the moment, but try to think about how that might work in an API (and Agents have had one for a very long time). I keep saying that Arctos has outgrown any single UI, eventually someone's going to believe me, THEN WHAT? I can't control their UI (nor do I want to, that's the point!), can some authorized user update an agent or can't they? (My take: there's no choice here, always assume that every user will have the ability to ignore any UI at will - because they usually do - but prioritize https://github.com/ArctosDB/arctos/issues/5172, which involves being able to know how 'curated' records have been edited and by whom, rolled back, etc.)

>make the code table request 

Seems right to me, but maybe someone/everyone remains unconvinced re: relationships??

**Comment by dustymc** [comment]
Tentatively calling this handled by https://github.com/ArctosDB/arctos/issues/7328, please reopen and elaborate if I'm confused on that.
